utils/run_exercise.py

```diff
@@ -307,8 +307,10 @@ def program_switches(self):
         for sw_name, sw_dict in self.switches.items():
             if 'cli_input' in sw_dict:
                 self.program_switch_cli(sw_name, sw_dict)
-            if 'runtime_json' in sw_dict:
+            elif 'runtime_json' in sw_dict:
```

Comment:
It appears that the original intent here is to allow the user to specify _both_ `cli_input` and `runtime_json` keys in the JSON file.  Do you see an issue with allowing both?

I understand that a user _could_ put conflicting commands in the two files, but I would consider that something that we simply warn them in documentation not to do, but the system won't catch them if they make a mistake.